tests/helpers/setup-ember-debug-test.js

```diff
@@ -51,6 +53,9 @@ export default function setupEmberDebugTest(hooks, options = {}) {
 
     this.owner.register('router:main', Router);
     this.owner.register('service:adapter', BasicAdapter);
+    this.owner.register('route:application', class extends ERouter {});
+    this.owner.register('controller:application', class extends Controller {});
+    this.owner.register('template:application', hbs('{{outlet}}'));
```

Comment:
I feel that these lines deserve a comment in the code, to help understand how the tests are structured. Something similar to what the PR description says, but phrased for someone who starts looking at this with no context.